drivers/dma/dma_stm32.c

```diff
@@ -452,7 +452,8 @@ DMA_STM32_EXPORT_API int dma_stm32_configure(const struct device *dev,
 	DMA_InitStruct.PeriphBurst = stm32_dma_get_pburst(config,
 							stream->source_periph);
 
-#if !defined(CONFIG_SOC_SERIES_STM32H7X) && !defined(CONFIG_SOC_SERIES_STM32MP1X)
+#if !defined(CONFIG_SOC_SERIES_STM32H7X) && !defined(CONFIG_SOC_SERIES_STM32H5X) && \
```

Comment:
Is this required for this PR?

Reply:
I will check that

Reply:
yes right , I check and the modifications in dma_stm32.c are not required